README.md

```diff
@@ -1,13 +1,68 @@
-# clouseau
+# Clouseau
 
 Clouseau uses Scalang to expose Lucene functionality via erlang-like nodes.
 
 ## Running a local dev cluster
 
 In separate terminal windows, run each of these commands;
 
+```
 mvn scala:run -Dlauncher=clouseau1
+```
 
+```
 mvn scala:run -Dlauncher=clouseau2
+```
 
+```
 mvn scala:run -Dlauncher=clouseau3
+```
+
+## Running in production
+
+Build Clouseau and move to `/opt/clouseau`:
+
+```
+cd clouseau-clone
```

Comment:
after git repo is cloned, the default directory name is `clouseau` instead of `clouseau-clone`.

Reply:
@jiangphcn Done!